### PR TITLE
added leak simulation and log parsing sections

### PR DESCRIPTION
**Pull request by ceremcem**
Added testing and log parsing sections. Related to #242 and #243.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,28 @@ using
...
+```
+tail /dev/zero
+```
+
+### Checking Logs 
+
+If you need any further actions after a process is killed by `earlyoom` (such as sending emails), you can parse the logs by:
+
+```
+sudo journalctl -u earlyoom | grep -i kill
```
Comment (jayenashar) [inline README.md:176]: this isn't listing everything for me.

`sudo journalctl -u earlyoom | grep -A1 sending` is what i'm using

File: README.md [hunk 2]
```diff
@@ -160,6 +160,28 @@ using
...
+```
+
+Example output for above test command (`tail /dev/zero`) will look like: 
+
+```
+Feb 19 18:21:39 hostnamehere earlyoom[15368]: Killing process 'tail' with signal 15, pid: 24833, badness: 972, VmRSS: 3963 MiB
```
Comment (jayenashar) [inline README.md:182]: i'm getting this:

```
Feb 20 10:59:34 eeyore earlyoom[10231]: sending SIGTERM to process 7378 uid 1000 "tail": badness 156, VmRSS 4962 MiB
Feb 20 10:59:34 eeyore earlyoom[10231]: process exited after 0.5 seconds
```

no kill.  earlyoom v1.6.2 on debian

**Comment by rfjakob** [comment]
I like this PR, thanks! But please update to the wording that latest earlyoom uses ( https://github.com/rfjakob/earlyoom/blob/e400b699d0fff156569758a2d5dad2a4fab60275/kill.c#L302 ) as @jayenashar said.

**Comment by ceremcem** [comment]
@rfjakob I agree too but I waited for your response. Should I replace that code something works for both versions or should I change to use latest version and add a separate note for older version?

**Comment by ceremcem** [comment]
It's actually easy if we ignore the "process exited after N seconds" line (which is not so important IMO). The code becomes: 

```
sudo journalctl -u earlyoom | grep -iE "(sending|killing)"
```

File: README.md [hunk 3]
```diff
@@ -160,6 +160,32 @@ using
...
+```
+Example output for above test command (`tail /dev/zero`) will look like: 
+
+```
+Feb 20 10:59:34 debian earlyoom[10231]: sending SIGTERM to process 7378 uid 1000 "tail": badness 156, VmRSS 4962 MiB
+```
+
+> For older versions of `earlyoom`, use: 
+> 
+>     sudo journalctl -u earlyoom | grep -iE "(sending|killing)"
```
Comment (jayenashar) [inline README.md:186]: you could add `-A1` to get the next line that tells you if the process did/didn't exit

```suggestion
>     sudo journalctl -u earlyoom | grep -A1 -iE "(sending|killing)"
```

**Review by rfjakob — APPROVED** [review]

**Pull request merged**